styleguide/derek/_partials/solutions/video.ejs

```diff
@@ -2,10 +2,12 @@
 <% if (!isSubpanel) { var isSubpanel = false; } %>
 
 <h2 class="videoEmbed-title">Video Title</h2>
-<div class="embed-responsive embed-responsive-16by9">
-  <% if (isSubpanel) { %>
-  <img class="embed-responsive-item" src="http://img.youtube.com/vi/duqBHik7nRo/maxresdefault.jpg"/>
-  <% } else { %>
-  <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/duqBHik7nRo"></iframe>
-  <% } %>
+<div class="videoEmbed-column">
```

Comment:
@writes, you don't have to add `videoEmbed-column` here.  This got removed from the pattern itself, and is now sitting *around* the pattern.  

See: https://github.com/rackerlabs/zoolander/blob/master/styleguide/derek/examples/solutions.ejs#L37